interp/constrintern.ml

```diff
@@ -359,13 +359,14 @@ let reset_hidden_inductive_implicit_test env =
          | (Inductive (params,_),b,c,d) -> (Inductive (params,false),b,c,d)
          | x -> x) env.impls }
 
-let check_hidden_implicit_parameters id impls =
+let check_hidden_implicit_parameters loc id impls =
```

Comment:
Would you mind using `?loc` here, as to be consistent with the optional status of a location.